mflowgen/full_chip/construct.py

```diff
@@ -21,7 +21,12 @@ def construct():
   #-----------------------------------------------------------------------
 
   adk_name = 'tsmc16'
-  adk_view = 'stdview'
+  adk_view = 'multivt'
+
+  # [steveri 11/2020] adk_view was: stdview. Now: multivt (above).
+  # Really only need 'multivt' for lvs (dragonphy uses lvt/ulvt)
+  # but I'm not sure the right way to do that
+  # so oh well everybody gets multivt...
```

Comment:
Ah, I see... it's to make LVS pass without hacking. I think that's reason enough. We should check the full_chip reports to see how many LVT/ULVT cells are used. I suspect it's not too many, but if it's a lot, we should probably come back to this.

Reply:
It's using a couple dozen LVT cells IIRC. But I have a better fix, as you probably have already noticed by now...